katz/src/main/kotlin/katz/predef.kt

```diff
@@ -0,0 +1,28 @@
+package katz
```

Comment:
package level objects in scala are usually lowercased and I carried that over but if this is not hipster approved in Kotlin I'll change it! in any case this file contains no instances or classes so a user will never interact with it directly. Now that you asked... What is the reason for camelcasing file names that contains no classes or instantiable objects and that are only pertinent to the compiler scope resolution? :trollface: